modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -48,6 +48,24 @@ def save_auto_claim!(auto_claim, status)
       auto_claim.save!
     end
 
+    def set_evss_response(auto_claim, error)
+      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
+
```

Comment:
@rockwellwindsor-va To clarify, are you suggewting to move line 52 out of this method, and add it to each rescue case? Thanks

Reply:
@stiehlrod a Since this is a shared method I think it works right there just the same, just thought it woud help to be before any of the other text stuff.